samples/retained-instance-store/src/main/java/com/badoo/ribs/samples/retained_instance_store/rib/RetainedInstanceRib.kt

```diff
@@ -0,0 +1,13 @@
+package com.badoo.ribs.samples.retained_instance_store.rib
+
+import com.badoo.ribs.core.Rib
+import com.badoo.ribs.samples.retained_instance_store.utils.Clock
+import com.badoo.ribs.samples.retained_instance_store.utils.ScreenOrientationController
+
+interface RetainedInstanceRib : Rib {
```

Comment:
Let's nest this under a parent to demonstrate disposal as well.

Reply:
Done